### PR TITLE
Create users table row

### LINKED ISSUE #453: Users table row
### Summary and Background
Users table row outlines basic information for an admin/volunteer.

### User Story
As Super Admin, I want to be able see an admin/volunteer's info in a clean and concise manner.

### Solution (Core Elements)
Create a row for the Users table.

Columns:
- Select box: On click, the row highlights purple and the box is ticked. Clicking again undos these changes.
- First name
- Last name
- Pronouns
- Email
- Phone number
- Three dots: On hover, 'View details' tooltip appears. On click, user profile side panel appears (#401).

### Mockups

Other designs: https://www.figma.com/file/TWJTNWjK75A6VgSisySTLe/Sistering-%2F-Admin-Experience?node-id=1643%3A48828

### Technical
- Create a new component for the user management table row under `frontend/src/components/admin/users/UserManagementTableRow.tsx`
- Each row should accept a prop for each column and a two props: checked for state management and onCheck handler
- The view details icon button should use the `MdMoreHoriz` icon from react-icons, see reference here: https://chakra-ui.com/docs/components/icon/usage#using-a-third-party-icon-library and https://react-icons.github.io/react-icons/search?q=more

### PR DESCRIPTION
**Pull request by twilkhoo**
## Ticket link

Closes #453

## Implementation description
* Created a table row using Chakra's `<Tr>`
* Added a sample ProfileDrawer to verify the view details functionality

image

## Steps to test
1. Login as admin and navigate to `/admin/users` and interact with the table, ensuring the row changes color when clicked
2. Click the three dots on the right and confirm the ProfileDrawer functionality

## What should reviewers focus on?
* In the code, please review how the `onCheck` functionality works with the table row, and the sample ProfileDrawer implementation.

## Checklist
- [x] My PR name is descriptive and in imperative tense
- [x] My commit messages are descriptive and in imperative tense. My commits are atomic and trivial commits are squashed or fixup'd into non-trivial commits
- [x] I have run the appropriate linter(s)
- [x] I have requested a review from the PL, as well as other devs who have background knowledge on this PR or who will be building on top of this PR

### CODE REVIEW

File: frontend/src/components/admin/users/UserManagementTableRow.tsx [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+import ProfileDrawer from "./ProfileDrawer";
+
+type UserManagementTableRowProps = {
+  firstName: string;
+  lastName: string;
+  pronouns: string;
+  email: string;
+  phoneNumber: string;
+  checked: boolean;
+  onCheck: (params: any) => any;
```
Comment (MatoPlus) [inline frontend/src/components/admin/users/UserManagementTableRow.tsx:22]: Can we make a TODO to change this type?
```suggestion
  // TODO: Replace param type
  onCheck: (params: any) => any;
```

File: frontend/src/components/admin/users/UserManagementTableRow.tsx [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+      </Td>
+      <Td>
+        <Text>{email}</Text>
+      </Td>
+      <Td>
+        <Text>{phoneNumber}</Text>
+      </Td>
+      <Td textAlign="right">
+        <Tooltip label="View details" placement="bottom-start">
+          <span>
```
Comment (MatoPlus) [inline frontend/src/components/admin/users/UserManagementTableRow.tsx:61]: Let's avoid spans, I think the alternative is a `Flex`

File: frontend/src/components/pages/admin/user/AdminUserManagementPage.tsx [hunk 3]
```diff
@@ -1,17 +1,17 @@
...
 import ErrorModal from "../../../common/ErrorModal";
 import Loading from "../../../common/Loading";
 import ProfileDrawer from "./ProfileDrawer";
 import { BranchResponseDTO } from "../../../../types/api/BranchTypes";
+import UserManagementTableRow from "../../../admin/users/UserManagementTableRow";
 
 const USERS = gql`
   query AdminUserManagementPage_Users {
     employeeUsers {
-      title
```
Comment (MatoPlus) [inline frontend/src/components/pages/admin/user/AdminUserManagementPage.tsx:14]: good catch!

**Review by MatoPlus — APPROVED** [review]
This was a lot more output than I expected, great initiative on staring on the entire table with concrete examples! LGTM with nits 🔥

File: frontend/src/components/admin/users/UserManagementTableRow.tsx (hunk 2 above)
Comment (twilkhoo) [reply frontend/src/components/admin/users/UserManagementTableRow.tsx]: Good idea, added a `<Box as="span">` instead, it's the equivalent.

**Review by MatoPlus — APPROVED** [review]
Ship it

**Pull request merged**